assets/css/sass/partials/_layout.scss

```diff
@@ -321,9 +321,6 @@ body {
     background-color: $secondary-color;
     color: white;
     display: none;
-    overflow: hidden;
```

Comment:
They were necessary to show an ellipsis when we were displaying the current location search specially, but aren't needed for other uses of `.location-search-alternate`.
![image](https://cloud.githubusercontent.com/assets/2162993/24100340/1e8eaa68-0d4a-11e7-8f5d-5904e42b095d.png)